aws/aws-codedeploy-agent
# Add configurable :time_zone for deployment logs timestamp

*Issue* #183

*Description of changes:*
* Introduce `:time_zone` config option which takes in either `local` or `utc` as option, fallback to `local` if not explicitly set
* Cast time to UTC ISO8601 format if `utc` option is chosen for deployment-log logger

*How to run test:*

`ruby -Itest test/instance_agent/log_test.rb`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
